### ISSUE TITLE
Changes in the file path text field don't register correctly

### ISSUE THREAD

**Issue by jmaglic** [jmaglic/MoloVol#50]
### Description

When writing anything in the radius and atom file text boxes, the changes don't register until after another text box has been selected.

### How to reproduce the issue

Browse and select an atom file. The file should load correctly. Afterwards, click into the atom file path text box and write something (doesn't matter where the text is added). Without clicking anything else, click directly on the "Reload" button. The atom file path will be reset and the valid file will be reloaded.

Clicking into the radius file path text box before clicking on "Reload" will keep the changes.

### Proposed change

Not sure how to solve this. Maybe an event should be caught when the text box is changed.

**Comment by rlavendomme** [comment]
The issue is not reproducible for me on windows 10.
Could you check on your windows device to know if it is a specific behavior of mac ?
Also, we would need someone with linux on the development team for testing purpose.

**Comment by jmaglic** [comment]
This problem wasn't limited to the file text box but applied to all TextCtrls. It was very annoying!

I've solved this issue in my most recent commit while reworking the UI. I fixed the undesired behaviour by catching the Event that is created when the text in any TextCtrl is changed and then changing the value in all TextCtrls to the current one. This is a bit circular, but it completely solves the issue.
